modules/send-training-data-to-models-bias-monitoring.adoc

```diff
@@ -0,0 +1,28 @@
+:_module-type: PROCEDURE
+
+[id="send-training-data-to-models-bias-monitoring_{context}"]
+= Sending training data to models
+
+In this section, you'll pass all the training data through the models to be able to compute baseline fairness values.
+
+. In a terminal window, run the following command from `2-BiasMonitoring`:
++
+[source]
+----
+for batch in 0 250 500 750 1000 1250 1500 1750 2000 2250; do
+  2-BiasMonitoring/scripts/send_data_batch
+2-BiasMonitoring/data/training/$batch.json
+done
+----
+
+This may take several minutes. The script prints out verification messages indicating whether TrustyAI is receiving the data, but you can also verify in the Cluster Metrics:
+
+. In the OpenShift web console, select *Observe* -> *Metrics*.
+. In the *Expression* field, enter `trustyai_model_observations_total` and click *Run Queries*. You should see both models listed, each reporting around 2250 observed inferences:
+. Optional step (time range isn't displayed until you run queries):
```

Comment:
It is not clear to me what "time range isn’t displayed until you run queries" means - are the following two lines the queries that you need to run??